### PR TITLE
Fix login

### PR DESCRIPTION
**Pull request by lucyxubroad**

### CODE REVIEW

File: Pear/Models/User.swift [hunk 1]
```diff
@@ -8,6 +8,24 @@
...
+    let facebook: String?
+    let firstName: String
+    let interests: [String]
+    let instagram: String?
+    let lastName: String
+    let major: String
+    let netID: String
+    let profilePictureURL: String?
+    let pronouns: String?
+
```
Comment (amycshuang) [inline Pear/Models/User.swift:26]: why are pronouns, hometown, and grad year optional if the user has to input them to onboard but major, name, groups, and interests are not?

File: Pear/Onboarding/DemographicsViewController.swift [hunk 2]
```diff
@@ -158,7 +158,7 @@ class DemographicsViewController: UIViewController {
                 case .value(let response):
                     if response.success {
                         let user = response.data
-                        if user.major != "" {
+                        if user.major != "" && user.major != "Undeclared" {
```
Comment (amycshuang) [inline Pear/Onboarding/DemographicsViewController.swift:161]: ```if user.major.isEmpty && user.major != "Undeclared"``` ?

**Review by amycshuang — APPROVED** [review]
looks good! :)

File: Pear/Onboarding/DemographicsViewController.swift (hunk 2 above)
Comment (lucyxubroad) [reply Pear/Onboarding/DemographicsViewController.swift:161]: Because backend is weird lol

File: Pear/Models/User.swift (hunk 1 above)
Comment (lucyxubroad) [reply Pear/Models/User.swift:26]: Just how backend structured this...

**Pull request merged**